src/dialogs/settingsdialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>800</width>
-    <height>600</height>
+    <width>1045</width>
```

Comment:
Please don't change any width and height values. 😅

Reply:
Sorry, it was the editor :), will revert these.